vMenu/menus/About.cs

```diff
@@ -28,7 +28,7 @@ private void CreateMenu()
             {
                 Label = $"~h~{MainMenu.Version}~h~"
             };
-            MenuItem credits = new MenuItem("About vMenu / Credits", "vMenu is made by ~b~Vespura~s~. For more info, checkout ~b~www.vespura.com/vmenu~s~. Thank you to: Deltanic, Brigliar, IllusiveTea, Shayan Doust and zr0iq for your contributions.");
+            MenuItem credits = new MenuItem("About vMenu / Credits", "vMenu is made by ~b~Vespura~s~. For more info, checkout ~b~www.vespura.com/vmenu~s~. Thank you to: Deltanic, Brigliar, IllusiveTea, Shayan Doust, zr0iq and DreamingCodes for your contributions.");
```

Comment:
Isn’t this something the resource maintainer is suppose to add if they feel the need to? Seems weird adding it yourself specially with the simplicity of this PR.

Reply:
I don't know what those people did to be there so I added myself if you don't evaluate my contribution enough to be there you are free to remove it =D.

Reply:
The reason those people were listed there is because they either helped extensively with testing in the early phases of development, or contributed significant amounts of code.

Other contributions are appreciated but don't usually end up in that list, they just show up on GitHub as contributors.